From d698cdefd69269bf6425cc576918aa3e2346492f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=23R=CE=BBatan?= <sharlatanus@gmail.com>
Date: Sun, 3 Jan 2021 21:45:07 +0000
Subject: [PATCH] Add new variables sbcl-db3 cl-db3 ecl-db3

---
 gnu/packages/lisp-xyz.scm | 27 +++++++++++++++++++++++++++
 1 file changed, 27 insertions(+)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 2d9ad8c7eb..b5622d57a3 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -9133,6 +9133,33 @@ foreign libraries.
                 " :ecl) \"usocket\""))
              #t)))))))
 
+(define-public sbcl-db3
+  (let ((commit "38e5ad35f025769fb7f8dcdc6e56df3e8efd8e6d")
+        (revision "1"))
+  (package
+    (name "sbcl-db3")
+    (version (git-version "0.0.0" revision commit))
+    (source (origin
+             (method git-fetch)
+             (uri (git-reference
+                   (url "https://github.com/dimitri/cl-db3")
+                   (commit commit)))
+             (file-name (git-file-name name version))
+             (sha256
+              (base32
+               "1i7j0mlri6kbklcx1lsm464s8kmyhhij5c4xh4aybrw8m4ixn1s5"))))
+    (build-system asdf-build-system/sbcl)
+    (home-page "https://github.com/dimitri/cl-db3")
+    (synopsis "Common Lisp library to read dbf files version 3")
+    (description "Allows processing data found in dbf and db3 files.")
+    (license license:public-domain))))
+
+(define-public ecl-db3
+  (sbcl-package->ecl-package sbcl-db3))
+
+(define-public cl-db3
+  (sbcl-package->cl-source-package sbcl-db3))
+
 (define-public sbcl-dbi
   ;; Master includes a breaking change which other packages depend on since
   ;; Quicklisp decided to follow it:
-- 
2.26.2

