From 94e19b2097f5d1ad20b928ccc421bddbdb8c265f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=23R=CE=BBatan?= <sharlatanus@gmail.com>
Date: Sat, 26 Dec 2020 23:56:07 +0000
Subject: [PATCH] Add new variables sbcl-uax-15 cl-uax-15 ecsl-uax-15

---
 gnu/packages/lisp-xyz.scm | 38 ++++++++++++++++++++++++++++++++++++++
 1 file changed, 38 insertions(+)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 885c4010a0..29af832d2a 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -455,6 +455,44 @@ compatible with ANSI-compliant Common Lisp implementations.")
 (define-public ecl-cl-ppcre
   (sbcl-package->ecl-package sbcl-cl-ppcre))
 
+(define-public sbcl-uax-15
+;"a3f17a053cf6f6091c44c8036a0b5fc171be6e23")
+  (let ((commit "d323188a69421d50f777bf86f024fb8b2b36e46f")
+         (revision "1"))
+  (package
+    (name "sbcl-uax-15")
+    (version (git-version "0.0.0" revision commit))
+    (source (origin
+              (method git-fetch)
+              (uri (git-reference
+                    (url "https://github.com/Hellseher/uax-15")
+                    ;(url "https://github.com/sabracrolleton/uax-15")
+                    (commit commit)))
+              (file-name (git-file-name name version))
+              (sha256
+               (base32
+                "0i2ykdaa5x2ph67bwxq6naai1x82msrn88ww0nvqr645hbb3jx66"))))
+    (build-system asdf-build-system/sbcl)
+    (native-inputs
+     `(("fiveam" ,sbcl-fiveam)))
+    (inputs
+     `(("split-sequence" ,sbcl-split-sequence)
+       ("cl-ppcre" ,sbcl-cl-ppcre)))
+    (arguments
+     `(#:tests? #f
+       #:asd-systems '("uax-15")))
+    (home-page "https://github.com/sabracrolleton/uax-15")
+    (synopsis "Common Lisp implementation of unicode normalization functions")
+    (description "This package supports UNICODE normalization and providing
+support for RFC8264 and RFC7564.")
+    (license license:expat))))
+
+(define-public cl-uax-15
+  (sbcl-package->cl-source-package sbcl-uax-15))
+
+(define-public ecl-uax-15
+  (sbcl-package->ecl-package sbcl-uax-15))
+
 (define-public sbcl-cl-unicode
   (package
     (name "sbcl-cl-unicode")
-- 
2.26.2

